specification/securityinsights/resource-manager/Microsoft.SecurityInsights/stable/2024-09-01/dataConnectorDefinitions.json

```diff
@@ -0,0 +1,706 @@
+    },
+    "ConnectorDataType": {
+      "description": "The data type which is created by the connector,\r\nincluding a query indicated when was the last time that data type was received in the workspace.",
+      "required": [
+        "lastDataReceivedQuery",
+        "name"
+      ],
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Gets or sets the name of the data type to show in the graph.",
+          "type": "string"
+        },
+        "lastDataReceivedQuery": {
+          "description": "Gets or sets the query to indicate when relevant data was last received in the workspace.",
+          "type": "string"
+        }
+      }
+    },
+    "DataConnectorDefinition": {
+      "allOf": [
+        {
+          "$ref": "../../../common/2.0/types.json#/definitions/ResourceWithEtag"
+        }
+      ],
+      "required": [
+        "kind"
+      ],
+      "description": "An Azure resource, which encapsulate the entire info requires to display a data connector page in Azure portal,\r\nand the info required to define data connections.",
+      "type": "object",
+      "properties": {
+        "kind": {
+          "$ref": "#/definitions/DataConnectorDefinitionKind",
+          "description": "The data connector kind",
+          "type": "string"
+        }
+      },
+      "discriminator": "kind"
+    },
+    "DataConnectorDefinitionKind": {
+      "description": "The kind of the data connector definitions",
+      "enum": [
+        "Customizable"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "modelAsString": true,
+        "name": "DataConnectorDefinitionKind",
+        "values": [
+          {
+            "value": "Customizable"
+          }
+        ]
+      }
+    },
+    "DataConnectorDefinitionArmCollectionWrapper": {
+      "type": "object",
+      "description": "Encapsulate the data connector definition object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DataConnectorDefinition"
+          }
+        },
+        "nextLink": {
+          "type": "string"
+        }
+      }
+    },
+    "CustomizableConnectorDefinition": {
+      "description": "Connector definition for kind 'Customizable'.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/DataConnectorDefinition"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/CustomizableConnectorDefinitionProperties",
+          "description": "Customizable properties.",
+          "x-ms-client-flatten": true
+        }
+      },
+      "x-ms-discriminator-value": "Customizable"
+    },
+    "CustomizableConnectorDefinitionProperties": {
+      "description": "The UiConfig for 'Customizable' connector definition kind.",
+      "type": "object",
+      "required": [
+        "connectorUiConfig"
+      ],
+      "properties": {
+        "createdTimeUtc": {
```

Comment:
>createdTimeUtc [](http://example.com/codeflow?start=9&length=14)

Use the top-level systemData properties instead of duplicating it in the property bag.